app/src/main/res/values/strings.xml

```diff
@@ -25,4 +25,8 @@ integration enabled on your home assistant instance.</string>
   <string name="pref_location_background_title">Background Location Tracking</string>
   <string name="pref_location_background_summary">Update your location behind the scenes, periodically.</string>
   <string name="application_version">Application Version</string>
+  <string name="location" tools:ignore="MissingTranslation">Location</string>
```

Comment:
Okay. Sorry to bring this up again. I think I am trying to understand 😅 

Is this error because the translations of this string is missing? In that case I think that we could just ignore it globally, because it will get uploaded to Lokalise and will get translated when the community does so.

Otherwise we wouldn't be able to add new languages either.

Reply:
That's what I understood it was, it will fail if any of the included languages doesn't have the string, right?

Reply:
Oh, in that case we can disable it globally, because as long as we have English and that's used as a fallback, we're good.

Reply:
I'll revert my revert. :-)

Reply:
Sorry!

Reply:
No problem, done.